src/storage/txn/commands/check_secondary_locks.rs

```diff
@@ -150,6 +151,7 @@ impl<S: Snapshot, L: LockManager> WriteCommand<S, L> for CheckSecondaryLocks {
             pr,
             lock_info: None,
             lock_guards: vec![],
+            response_policy: ResponsePolicy::OnApplied,
```

Comment:
How about making it a method of `CommandExt`?

Reply:
Sorry I didn't understand 😥

Reply:
@youjiali1995  Currently a command itself cannot read configs of the scheduler, so I think it will be a bit difficult to do it.

Reply:
Ah, I got it